content/zh/docs/tasks/security/authentication/authn-policy/index.md

```diff
@@ -504,6 +508,33 @@ spec:
 EOF
 {{< /text >}}
 
+{{< /tab >}}
+
+{{< tab name="Gateway API" category-value="gateway-api" >}}
```

Comment:
I think this contains some dup work compared to https://github.com/istio/istio.io/pull/13646

Reply:
Yes, I will merge resolve all after the #13646 finished.